lib/bin/supllib/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if(CONFIG_FLOAT)
+  if(CONFIG_FP_HARDABI)
+    set(float_dir hard-float)
+  else()
+    assert(0 "Unreachable code")
```

Comment:
Is this unreachable code? Could the user not have enabled CONFIG_FP_SOFTABI ?

Reply:
we don't support it, the SUPL library is only built for hard-float.
But maybe the message instead should have been "CONFIG_FP_SOFTABI not supported"

Reply:
soft-float is given by not enabling CONFIG_FLOAT. But "Soft float not supported" is not the right error message
because FP_SOFTABI is not soft-float.

The message should be changed to "CONFIG_FP_SOFTABI is not supported".

Reply:
Changed the message text, you will not this update as I moved the CMakeLists.txt file into the zipfile that user download.